git/github/github.go

```diff
@@ -36,6 +36,7 @@ type CheckRunAdapter struct {
 	Name           string
 	SHA            string
 	ExternalID     string
+	DetailsURL     string
```

Comment:
Same as below,  I think this value should be set to a formatted pipelineURL somewhere in the code.

Reply:
I've added an assignment with console URL link with  TestPipelineRunName  [here](https://github.com/kasemAlem/integration-service/blob/32590e8a4646c8800a029fffb1125763b3a6f939/status/reporter_github.go#L181)